client/wfnews-war/src/main/angular/src/app/components/report-of-fire/complex-question-page/rof-complex-question-page.component.ts

```diff
@@ -113,4 +113,10 @@ export class RoFComplexQuestionPage extends RoFPage {
   backToReview() {
     this.reportOfFirePage.edit('review-page')
   }
+
+  previousPage() {
+    if (this.id === 'distance-page') this.reportOfFire.headingDetectionActive = true;
```

Comment:
You could probably put this in an override on the distance page itself, rather than adding logic into the complex question page